public/dashboard.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <p id="welcome-text">welcome to your diary</p>
+      <button id="btnLogout">Logout</button>
+    </header>
+
+    <section class="create-entry">
+      <form id="formCreate">
+        <input type="text" id="title" placeholder="title" />
+        <textarea
+          name="make-entry"
+          id="body"
+          placeholder="Write about your day"
+        ></textarea>
+        <button type="submit">Post</button>
```

Comment:
why doesn't you button have a name?

Reply:
dont forget that we are responsible for that he was preparing the code for what we're bringing

Reply:
Does not need. It's job is simply to trigger the submit action on the form.

Reply:
Oh, noted, I wanted to be sure so that we don't leave it out mate. Got it. So the the js function on form submit, I get it